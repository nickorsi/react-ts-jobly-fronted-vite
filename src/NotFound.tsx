import React from "react";

function NotFound() {
    return (
        <>
            <h2>Sorry!</h2>
            <p>Looks like that page doesn't exist!</p>
            <p>Instead why don't you look at the rest of our site!</p>
        </>
    )
}

export default NotFound;