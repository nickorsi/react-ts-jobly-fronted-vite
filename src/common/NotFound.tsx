import "./NotFound.css";

/**
 * NotFound componenet with generic message for users that have reached a URL
 * that is not mapped to a specific path.
 *
 * RoutesList -> NotFound
 */
function NotFound() {
    return (
        <div className="not-found card">
            <h2>Sorry! ðŸ˜…</h2>
            <p>Looks like that page doesn't exist!</p>
            <p>Please check out the rest of our site with the links above!</p>
        </div>
    )
}

export default NotFound;