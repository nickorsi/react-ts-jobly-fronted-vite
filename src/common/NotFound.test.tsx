import {describe, expect, it} from "vitest";
import { render } from "@testing-library/react";

import NotFound from "./NotFound.tsx";

describe("Simple Render Tests", () => {
    it("renders without crashing", function () {
        render(<NotFound />);
    });
    it("matches the snapshot", function () {
        const { container } = render(
            <NotFound />
        );

        expect(container).toMatchSnapshot();
    });
});