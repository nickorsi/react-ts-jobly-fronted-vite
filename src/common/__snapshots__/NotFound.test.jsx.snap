// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Simple Render Tests > matches the snapshot 1`] = `
<div>
  <div
    class="not-found card"
  >
    <h2>
      Sorry! ðŸ˜…
    </h2>
    <p>
      Looks like that page doesn't exist!
    </p>
    <p>
      Please check out the rest of our site with the links above!
    </p>
  </div>
</div>
`;
